src/components/measureLanding/measureList/actionCenter/shareAction/ShareAction.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback, useEffect, useState } from "react";
+import { IconButton } from "@mui/material";
+import Tooltip from "@mui/material/Tooltip";
+import { Measure } from "@madie/madie-models";
+import DeleteOutlinedIcon from "@mui/icons-material/DeleteOutlined";
+import { grey, red } from "@mui/material/colors";
+import ShareIcon from "../../../../common/ShareIcon";
+
+interface PropTypes {
+  measures: Measure[];
+  canEdit: boolean;
+}
+
+export const NOTHING_SELECTED = "Select measure to share";
+export const SHARE_MEASURE = "Share measure";
+
+export default function ShareAction(props: PropTypes) {
+  const { measures, canEdit } = props;
+  const [disableShareBtn, setDisableShareBtn] = useState(true);
+  const [tooltipMessage, setTooltipMessage] = useState(NOTHING_SELECTED);
+
+  const validateShareActionState = useCallback(() => {
+    setDisableShareBtn(true);
+    setTooltipMessage(NOTHING_SELECTED);
```

Comment:
I think this tooltip would be a lil bit confusing if more than one measure is selected. it would still display nothing selected.

Reply:
I have added this message based on the story ACs